doc/proposals/ansible-operator-webhooks.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Ansible Operator Webhooks
+authors:
+  - "@fabianvf"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2019-10-14
+last-updated: 2019-10-14
+status: provisional
+---
+
+# Ansible Operator Webhooks
+
+## Release Signoff Checklist
+
+- \[ \] Enhancement is `implementable`
+- \[ \] Design details are appropriately documented from clear requirements
+- \[ \] Test plan is defined
+- \[ \] Graduation criteria for dev preview, tech preview, GA
+- \[ \] User-facing documentation is created in [operator-sdk/doc][operator-sdk-doc]
+
+## Open Questions (optional)
+
+- For non-trivial playbooks, will the performance be acceptable?
+- How much webhook configuration should we support?
+- Should we support side-effects?
+    Side-effects will require adding reconciliation mechanisms to the webhook,
+    and would likely impact the performance even more extremely. My instinct is to not support it as it adds
+    a lot of complexity and seems dangerous, but then we're obviously restricting the power of the webhooks.
+    If we don't allow side-effects, we could enforce that by making the proxy reject requests from the webhooks
+    that do anything but `GET`.
+- Need to determine exactly what information we should send to the webhooks, and how it will be structured.
+    Passing this information to the Runner will require some minor refactors.
+- Where should artifacts for the webhook runs end up?
+
```

Comment:
If it has nothing to be added here then we should remove the section. 
However, may the timeout subject could be something raised here. WDYT?